packages/squiggle-lang/src/rescript/Utility/Operation.res

```diff
@@ -96,12 +119,27 @@ module DistToFloat = {
 // Note that different logarithms don't really do anything.
 module Scale = {
   type t = scaleOperation
-  let toFn = x =>
+  let toFn = (x: t, a: float, b: float): result<float, invalidOperationError> =>
     switch x {
-    | #Multiply => \"*."
-    | #Divide => \"/."
-    | #Power => \"**"
-    | #Logarithm => (a, b) => log(a) /. log(b)
+    | #Multiply => Ok(a *. b)
+    | #Divide =>
```

Comment:
Is this duplicating code that's in Algebraic?

Reply:
Very good question, let me check